frontend/src/queries/nodes/WebVitals/WebVitalsProgressBar.tsx

```diff
@@ -28,9 +28,9 @@ export function WebVitalsProgressBar({ value, metric }: WebVitalsProgressBarProp
                 // eslint-disable-next-line react/forbid-dom-props
                 style={{
                     width: `${goodWidth}%`,
-                    backgroundColor: band === 'good' ? WEB_VITALS_COLORS.good : undefined,
+                    backgroundColor: band === 'good' ? `${WEB_VITALS_COLORS.good} !important` : undefined,
```

Comment:
I'm not quite sure why these changes are included in this PR, but I trust you to verify whether they are intentional.

Reply:
Yep, that was a fly-by